c/array/array_rotate.c

```diff
@@ -0,0 +1,62 @@
+
+#include <stdbool.h>
+#include <assert.h>
+#include <time.h>
+#include "array_utils.h"
+
+#define MAX_SIZE 10
+
+/*
+ *rotate array rotaates the given original array for the given
+ *number of times.
+ *for example 1 2 3 4 5 6 7 8 9 10 is the original array
+ *then it roates it to 9 10 1 2 3 4 5 6 7 8 when number of times
+ *rotate is 2.
+*/
+
```

Comment:
remove this empty line. The function comment should immediately preceed the function defintiion.
Do it for other comments also.